react-charts/area-chart/intro.md

```diff
@@ -1 +1,15 @@
-In this scenario, you will learn how to use PatternFly React area chart components for the first time.
+In this scenario, you will learn how to use PatternFly React area chart components.
+
+You will begin by adding a CSS selector to define the chart size...
+
+<strong>Note:</strong> Katacoda is setting up a new React application for you. You will be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
```

Comment:
Let's make this a tad more concise with a few fast (and fun) tweaks: Katacoda is setting up a new React application. You can use the code once the server starts and you can see "Welcome to PatternFly" on the lower pane.